integration-tests/src/test/groovy/com/okta/sdk/tests/it/EventHooksIT.groovy

```diff
@@ -0,0 +1,254 @@
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.resource.ResourceException
+import com.okta.sdk.resource.EventHookChannel
+import com.okta.sdk.resource.EventHookChannelConfig
+import com.okta.sdk.resource.EventHookChannelConfigAuthScheme
+import com.okta.sdk.resource.EventHookChannelConfigHeader
+import com.okta.sdk.resource.EventSubscriptions
+import com.okta.sdk.resource.event.hook.EventHook
+import com.okta.sdk.resource.event.hook.EventHookChannelConfigAuthSchemeType
+import com.okta.sdk.resource.event.hook.EventHookList
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.annotations.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.equalTo
+import static org.hamcrest.Matchers.hasSize
+import static org.hamcrest.Matchers.notNullValue
+
+/**
+ * Tests for /api/v1/eventHooks
+ * @since 2.0.0
+ */
+class EventHooksIT extends ITSupport {
+
+    @Test
+    void createEventHookTest() {
+        String name = "event-hook-java-sdk-${UUID.randomUUID().toString()}"
+
+        EventHook createdEventHook = client.instantiate(EventHook)
+            .setName(name)
+            .setEvents(client.instantiate(EventSubscriptions)
+                .setType(EventSubscriptions.TypeEnum.EVENT_TYPE)
+                .setItems(["user.lifecycle.create", "user.lifecycle.activate"]))
+            .setChannel(client.instantiate(EventHookChannel)
+                .setType(EventHookChannel.TypeEnum.HTTP)
+                .setVersion("1.0.0")
+                .setConfig(client.instantiate(EventHookChannelConfig)
+                    .setUri("https://www.example.com/eventHooks")
+                    .setHeaders([ client.instantiate(EventHookChannelConfigHeader)
+                                      .setKey("X-Test-Header")
+                                      .setValue("Test header value")])
+                    .setAuthScheme(client.instantiate(EventHookChannelConfigAuthScheme)
+                .setType(EventHookChannelConfigAuthSchemeType.HEADER)
+                .setKey("Authorization")
+                .setValue("Test-Api-Key"))))
+        registerForCleanup(createdEventHook)
+
+        assertThat(createdEventHook.getId(), notNullValue())
+        assertThat(createdEventHook.getName(), equalTo(name))
+        assertThat(createdEventHook.getEvents(), hasSize(2))
+        assertThat(createdEventHook.getChannel().getConfig().getUri(), equalTo("https://www.example.com/eventHooks"))
+    }
+
+    @Test
+    void getEventHookTest() {
+        String name = "event-hook-java-sdk-${UUID.randomUUID().toString()}"
+
+        EventHook createdEventHook = client.instantiate(EventHook)
+            .setName(name)
+            .setEvents(client.instantiate(EventSubscriptions)
+                .setType(EventSubscriptions.TypeEnum.EVENT_TYPE)
+                .setItems(["user.lifecycle.create", "user.lifecycle.activate"]))
```

Comment:
Are these defined somewhere? An enum probably doesn't fit for these given that they will change over time, but maybe we could define them as static args somewhere? 🤔

Reply:
Sure, will add a static constant.